tests/e2e/test_deploying.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+@pytest.mark.usefixtures("init")
+def test_deploying_basic_example(protostar):
+    protostar(["build"])
+
+    result = protostar(["deploy", "./build/main.json", "--network", "alpha-goerli"])
```

Comment:
Can we test in on a contract with constructor arguments? We sould also deploy it on devnet. Also please provide a test case for configuration: contract with constructor arguments using config file